src/sota_tools/treehub_server.cc

```diff
@@ -35,8 +44,32 @@ void TreehubServer::InjectIntoCurl(const string& url_suffix, CURL* curl_handle)
     curl_easy_setopt(curl_handle, CURLOPT_PASSWORD, password_.c_str());
   }
 
-  if (ca_certs_ != "") {
-    curl_easy_setopt(curl_handle, CURLOPT_CAINFO, ca_certs_.c_str());
+  boost::filesystem::path tmp_dir = boost::filesystem::temp_directory_path() / boost::filesystem::unique_path();
```

Comment:
Leaving junk around in a user's tmp directory isn't great--can't we use `TemporaryFile` from utils.h to manage this for us (but we'll have to make the TemporaryFile a member variable of TreehubServer since the curl handle outlives this function call.

Reply:
Good Idea